web/src/frontend/classic/stores/Documentation.ts

```diff
@@ -0,0 +1,80 @@
+import { writable } from 'svelte/store';
+
+export type DocCategory = {
+    name: string;
+    description: HTMLElement;
+    sections: DocSection[];
+}
+export type DocSection = {
+    name: string,
+    content: HTMLElement
+}
+
+export const documentation = writable<DocCategory[]>([]);
+
+/**
+ * Creates a slide section by extracting HTML elements from the given content element until a specified split element is encountered.
+ * @param content - The content element from which to extract the slide section.
+ * @param split - The element name at which to stop extracting and split the slide section.
+ * @returns The created slide section element.
+ */
+function slideElement(content:HTMLElement, split:string[]) : HTMLElement {
```

Comment:
Readability can be improved by adding space between parameter name and type declaration `content: HTMLElement`.